upf/upf_pfcp_api.c

```diff
@@ -147,6 +149,41 @@ init_tp_error_report (pfcp_tp_error_report_t * report,
 
 /* message helpers */
 
+static void
+build_ue_ip_address_information (pfcp_ue_ip_address_pool_information_t **
+				 ue_pool_info)
+{
+  upf_main_t *gtm = &upf_main;
+  upf_nat_pool_t *np;
+  upf_ueip_pool_info_t *ue_p;
+
+  vec_alloc (*ue_pool_info, pool_elts (gtm->ueip_pools));
+
+  pool_foreach (ue_p, gtm->ueip_pools)
+  {
+    pfcp_ue_ip_address_pool_information_t *ueif;
+    vec_add2 (*ue_pool_info, ueif, 1);
```

Comment:
there should be empty line after the local variable definition

Reply:
done